wrappers/python/simtk/openmm/app/forcefield.py

```diff
@@ -86,18 +86,42 @@ def _parseFunctions(element):
                 params[key] = int(function.attrib[key])
             elif key.endswith('min') or key.endswith('max'):
                 params[key] = float(function.attrib[key])
+        if functionType.startswith('Continuous'):
+            params['periodic'] = eval(function.attrib.get('periodic', 'False').title())
```

Comment:
This is dangerous.  It will execute any Python code put into the attribute.  A better way of doing it is

```python
params['periodic'] = (function.attrib.get('periodic', 'false').lower() == 'true')
```

Reply:
If the user makes a mistake when trying to enter `true`, this will evaluate to `false` without notice. How about the alternative below?

```py
        if functionType.startswith('Continuous'):
            periodicStr = function.attrib.get('periodic', 'false').lower()
            if periodicStr in ['true', 'false', 'yes', 'no', '1', '0']:
                params['periodic'] = periodicStr in ['true', 'yes', '1']
            else:
                raise ValueError('ForceField: non-boolean value for periodic attribute in tabulated function definition')
```